### ISSUE TITLE
Insufficient alt text on component pages

### ISSUE THREAD

**Issue by adorayi** [cds-snc/design-gc-conception#681]
https://docs.google.com/spreadsheets/d/1FHBEau_Bep6C_FU2Duf58MbDUlY6v_5Bjm9kEGSplaA/edit#gid=0 

- [x] Update alt text
- [x] Translate
- [ ] Publish changes
